docs/docs-ref-autogen/excel/excel/excel.range.yml

```diff
@@ -1879,10 +1882,10 @@ methods:
     package: excel!
     fullName: getImage()
     summary: >-
-      Renders the range as a base64-encoded png image. **Important**: This API
-      is currently unsupported in Excel for Mac. Visit [OfficeDev/office-js
-      Issue \#235](https://github.com/OfficeDev/office-js/issues/235) for the
-      current status.
+      Renders the range as a base64-encoded png image. *Important**: This API is
```

Comment:
```suggestion
      Renders the range as a base64-encoded png image. **Important**: This API is
```

Reply:
@alison-mk @Rick-Kirkham You'll need to update this in DefinitelyTyped